sssom/cli.py

```diff
@@ -60,14 +60,14 @@
 output_option = click.option(
     "-o",
     "--output",
-    help="Output file, e.g. a SSSOM tsv file.",
+    help="Path of SSSOM output file.",
     type=click.File(mode="w"),
     default=sys.stdout,
 )
 output_format_option = click.option(
     "-O",
     "--output-format",
-    help=f'Desired output format, e.g. {",".join(SSSOM_EXPORT_FORMATS)}',
+    help=f'Desired output format, e.g. {",".join(SSSOM_EXPORT_FORMATS + ["fhir"])}',
```

Comment:
why not add fhir to SSSOM_EXPORT_FORMATS?

Reply:
donezo